aikau/src/main/resources/alfresco/documentlibrary/_AlfDocumentListTopicMixin.js

```diff
@@ -178,6 +178,9 @@ define(["dojo/_base/declare",
        * @instance
        * @type {string} 
        * @default
+       * @event
+       * @property {string} [value] One of "selectAll", "selectNone", "selectInvert", "selectFolders" or "selectDocuments"
+       * @property {object[]} [selectedItems] An array of items that have been selected
```

Comment:
Populate the @default value to be the topics.XXX event
Keep the @event tag
Lose the @property tags